stumpy/scrump.py

```diff
@@ -355,8 +405,13 @@ def prescrump(T_A, m, T_B=None, s=None, normalize=True, p=2.0):
         indices,
         s,
         excl_zone,
+        k,
     )
 
+    if k == 1:
```

Comment:
Because I did not want to change it for users who already used this module. This is a public API. If I do not do `.ravel()`, it will have shape `(l, 1)`. We do not have to do it. But, I thought it might be better to keep the structure for this pubic function.

Reply:
I see. Hmmm, this is a tough one. The fact of the matter is that `prescrump` isn't actually listed in our API so while it is technically "public" based on the way the function is named, I'd say that the function is really still "private" from public knowledge/use and that it's probably safe to change it to a 2D array for our purpose. Of course, this means that you'll have to modify `scrump` accordingly since it calls `prescrump` and expects a 1D array being returned.

Reply:
I got your point. So, I will remove this `if-block` and allows shape `(l, 1)` when `k` is 1 for `prescrump`. And, I will try to keep that in mind for `scrump`.